distributed/scheduler.py

```diff
@@ -477,7 +477,7 @@ def __init__(
         self.versions = versions or {}
         self.nanny = nanny
         self.status = status
-        self._hash = hash(address)
+        self._hash = hash(address) + hash(pid) + hash(name)
```

Comment:
```suggestion
        self._hash = hash((address, pid, name))
```

Reply:
Here.

I really don't care though, and I'm inclined just to press the green button in order to avoid the CI run and move along.

Reply:
done